docs/remote_inference_blueprints/cohere_v2_connector_embedding_blueprint.md

```diff
@@ -22,7 +22,8 @@ POST /_plugins/_ml/connectors/_create
       "method": "POST",
       "url": "https://api.cohere.ai/v1/embed",
       "headers": {
-        "Authorization": "Bearer ${credential.cohere_key}"
+        "Authorization": "Bearer ${credential.cohere_key}",
+        "Request-Source": "unspecified:opensearch"
```

Comment:
Is that a mandatory field? You probably also want to make sure you update all the tests as well to reflect on that. For example:
https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

Reply:
From the documentation looks like its not. But since we are doing it anyway lets update all the places do have ?

Reply:
Thanks for adding this. Can you explain what does `unspecified` mean ?

Reply:
`unspecified` just means when we log in analytics the request-source is not from any of our supported sdks

Reply:
Got it , sounds this format `unspecified:opensearch` means `{sdk}:{source}` ? looks something coupled with implementation of your side ?

Reply:
> if yes, I think we should not add this part to other blueprints which are not for Cohere models

Sure, I meant to say just the cohere API's. Fixed my comment.

Reply:
yeah haha, just the Cohere APIs. Thank you